libs/test-utils/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@ledgerhq/test-utils",
+  "version": "0.0.1",
```

Comment:
private: true

Reply:
`test-utils:dummy-wallet`
`test-utils:dummy-live`

for shortcuts in the root package.json